tests/ballerina-unit-test/src/test/resources/test-src/types/table/table_type.bal

```diff
@@ -124,7 +124,10 @@ function testToJson(string jdbcUrl, string userName, string password) returns (j
     try {
         table dt = check testDB->select("SELECT int_type, long_type, float_type, double_type,
                   boolean_type, string_type from DataTable WHERE row_id = 1", ());
-        return check <json>dt;
+        json result = check <json>dt;
+        // Converting to string to make sure the json is built before returning.
+        _ = result.toString();
+        return result;
```

Comment:
Why we need to do this conversion?

Reply:
Otherwise when the result is checked at the java side, this function would have already returned so the result won't be streamed since we have stopped the endpoint. This is done just to touch the json and build it before function returns.